docs/configure_admin_workstation_post_install.rst

```diff
@@ -39,7 +39,7 @@ with these scripts (``~/Persistent/securedrop/``), and run the install script:
 
     ./securedrop-admin tailsconfig
 
-Type the Administration Password that you selected when starting Tails and hit
+Type the Administration Passphrase that you selected when starting Tails and hit
```

Comment:
This one we must keep because it is a reference to tails as shown in the greeter screen below.

![a](https://user-images.githubusercontent.com/433594/33703068-bb0de4a8-db26-11e7-8fc3-c21a0a332a4c.png)

Reply:
Ah good catch :)